### DISCUSSION TITLE
Installing for Windows

### DISCUSSION THREAD

**Discussion by JJLewin** [dvehrs/podget#68] [General]
Hello. Since I am not a programmer /tekkie, I was interested in whether, and how, podget could be installed on a Windows 10 OS.

**Comment by dvehrs** [comment]
Good Afternoon,

I knew this day would someday arrive.  We've gotten Podget working on almost every other OS so eventually someone had to ask about Windows and here we are.  Unfortunately I don't have a simple answer for you.  The last version of Windows I had on one of my PCs was Windows 98SE but I have a few guesses on how it could be done.  

I do most of the development for Podget on Debian Linux so that is where I would start.  The Debian wiki has a short page on how to install Debian Linux into the Windows Subsystem for Linux on Windows 10.

https://wiki.debian.org/InstallingDebianOn/Microsoft/Windows/SubsystemForLinux

Once you have that working, then you will need to run "apt-get update && apt-get upgrade" to make sure everything is updated to latest versions.  Might also be good to check what version of Debian it has installed and upgrade to a newer version if necessary (how to do that is in the FAQ section of the page above).  

Then we can install Podget with "apt-get install podget".

NOTE:  Podget is not a GUI app so you won't have to follow those steps, but you can if you like.  It may be helpful to have some simple GUI text editors for the config files.  Otherwise you will need to learn how to use a console based editor line nano, vim or ed (probably stay away from ed unless you want a real challenge).    

Now all of that is just a guess on how it can be done.  I do not have a working version of Windows 10 to test it on myself.  In theory this will all work but it is untested.  

What do you think, is that something you would consider trying?  If it does work, we can figure out where to add some installation notes for other users.

Dave

**Comment by benpollinger** [comment]
I tried this on my Windows 10 machine with Debian as a subsystem. All worked fine with apt-get (or apt) install podget after the apt update and upgrade.

Then I ran podget without making and config changes and it pulled the latest The Linux Link episode and added it to a playlist.

Debian's home dir is under C:\Users\%username%\AppData\Local\Packages\TheDebianProject.DebianGNULinux_76v4gfsz19hv4\LocalState\rootfs\home\ though I guess that string after 'DebianGNULinux_' might vary.

**Comment by dvehrs** [comment]
Thanks Ben!

Maybe not the easiest to use on Windows but it is good to know it works.   I will be converting this issue to a discussion so that it is never archived and people can find it.
